results/results.js

```diff
@@ -1,33 +1,263 @@
-totalPicks.push(localPicks);
-totalDisplay.push(localDisplay);
+// push the data from this session onto totals array
+localPicks.forEach(pick => {
+    totalPicks.push(pick);
+});
 
-sessionPicked.textContent = localPicks;
-sessionDisplayed.textContent = localDisplay;
-
-totalDisplayed.textContent = totalDisplay;
-totalPicked.textContent = totalPicks;
+localDisplay.forEach(display => {
+    totalDisplay.push(display);
+});
 
+// set the totals into local storage
 let storeTotalPicks = JSON.stringify(totalPicks);
 localStorage.setItem('TOTALPICKS', storeTotalPicks);
 
 let storeTotalDisplay = JSON.stringify(totalDisplay);
 localStorage.setItem('TOTALDISPLAYED', storeTotalDisplay);
 
+function findById(items, id) {
+    // loop the array
+    for (let i = 0; i < items.length; i++) {
+        const item = items[i];
+
+        // check the id against item.id
+        if (item.id === id) {
+            return item;
+        }
+    }
+
+    // loop done, nothing found
+    return null;
+}
+
+// putting the data into a more easily useable format
+const sessionVotes = [];
+const totalVotes = [];
+
+function addInitialVoteItem(votes, id) {
+    const voteItem = {
+        id,
+        timesSeen: 0,
+        timesPicked: 0
+    };
+    votes.push(voteItem);
+}
+
+function incrementTimesSeen(votes, id) {
+    let temp = findById(votes, id);
+    if (!temp) {
+        addInitialVoteItem(votes, id);
+        temp = findById(votes, id);
+    }
+    temp.timesSeen++;
+}
+
+function incrementTimesPicked(votes, id) {
+    let temp = findById(votes, id);
+    if (!temp) {
+        addInitialVoteItem(votes, id);
+        temp = findById(votes, id);
+    }
+    temp.timesPicked++;
+}
+
+// loop through local picks in local storage and push each time item is picked
+localPicks.forEach(pick => {
+    incrementTimesPicked(sessionVotes, pick);
+});
+
+// loop through local picks in local storage and push each time item is displayed
+localDisplay.forEach(display => {
+    incrementTimesSeen(sessionVotes, display);
+});
+
+// same thing as session but for total
+totalPicks.forEach(pick => {
+    incrementTimesPicked(totalVotes, pick);
+});
+
+totalDisplay.forEach(display => {
+    incrementTimesSeen(totalVotes, display);
+});
+
+// session info for graphs
+const graphIds = sessionVotes.map(el => el.id);
+const graphSessionPicked = sessionVotes.map(el => el.timesPicked);
+const graphSessionSeen = sessionVotes.map(el => el.timesSeen);
+
+var ctx = document.getElementById('session-results-picked').getContext('2d');
+new Chart(ctx, {
+    type: 'bar',
+    data: {
+        labels: graphIds,
+        datasets: [{
+            label: '# of Votes',
+            data: graphSessionPicked,
+            backgroundColor: [
+                'rgba(255, 99, 132, 0.2)',
+                'rgba(54, 162, 235, 0.2)',
+                'rgba(255, 206, 86, 0.2)',
+                'rgba(75, 192, 192, 0.2)',
+                'rgba(153, 102, 255, 0.2)',
+                'rgba(255, 159, 64, 0.2)'
```

Comment:
Your bars are showing up as grey because you haven't defined colors for them here :)